spec/fake_braintree/credit_card_spec.rb

```diff
@@ -77,6 +77,13 @@ def token_for(month, year)
     end
   end
 
+  it "sets the creation time" do
+    credit_card = Braintree::CreditCard.create(build_credit_card_hash).credit_card
+
+    creation_time = Time.parse(credit_card.created_at)
+    expect(creation_time).to be_within(1).of(Time.now)
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.